packages/gatsby-design-tokens/src/index.js

```diff
@@ -0,0 +1,33 @@
+import borders from "./borders"
```

Comment:
please use the export syntax so we can do treeshaking.
```suggestion
export { default as borders } from "./borders"
```

Reply:
Done (I think 😬): 47ab9d8